src/design-guidelines/color-system/examples/contrast-color-palette.scss

```diff
@@ -1,12 +1,6 @@
 @use '../../../style/functions';
```

Comment:
The colour of the vertical ellipsis in split-button has changed.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212076891-070911a3-0166-47bd-8e3b-3df26b92ee58.png">

(NOT a show-stopper. Fix _after_ merge please 🤗)